cg/meta/backup.py

```diff
@@ -53,21 +54,22 @@ def fetch_flowcell(self, flowcell_obj: models.Flowcell = None, dry: bool = False
             LOG.info("maximum flowcells ondisk reached")
             return None
 
-        if flowcell_obj is None:
+        if not dry_run and flowcell_obj is None:
             flowcell_obj = self.pop_flowcell()
             if flowcell_obj is None:
                 LOG.info("no flowcells requested")
                 return None
 
-        LOG.info("%s: retreiving from PDC", flowcell_obj.name)
+        if not dry_run:
```

Comment:
Why would we not log this even if it is a dryrun?

Reply:
Fixed here https://github.com/Clinical-Genomics/cg/pull/397/commits/e272b84fed676d2d759b0f83e2dac589050b8d04